src/Altinn.AccessManagement/appsettings.Development.json

```diff
@@ -32,6 +32,7 @@
     "SecretUri": "http://devenv.altinn.no/"
   },
   "FeatureManagement": {
-    "RightsDelegationApi": true
+    "RightsDelegationApi": true,
+    "UseNewQueryRepo": false
```

Comment:
Default verdi for UseNewQueryRepo må vel inn i appsettings.json skal den også fungere for deploy til alle miljøer? Ellers må den vel inn i values.yaml filen

Reply:
`default(bool)` er `false`, saa teknisk sett saa er det redundant med `GetValue<bool>("name", defaultValue: false)` - men er definitvt med eksplisit.